.github/workflows/go-tests.yml

```diff
@@ -47,6 +47,12 @@ jobs:
           EOF
           DEBIAN_FRONTEND=noninteractive sudo apt-get install -y --force-yes couchdb=${{ matrix.couchdb-version }}*
           echo "COZY_COUCHDB_URL=http://admin:password@localhost:5984/" >> $GITHUB_ENV
+      
+        # This step is required to let ImageMagick generate PDF files.
+      - name: Fix ImageMagick config
+        run: |
+          sudo sed -i '/disable ghostscript format types/,+6d' /etc/ImageMagick-6/policy.xml
+          cat /etc/ImageMagick-6/policy.xml
```

Comment:
I think the `cat` here is for debugging, right?

Reply:
Yep, this is still a WIP but I can't find a way to move back a PR into a draft.

BTW it seems that there is some other libs missing. I also test the commit on #3796 in order to check what it does when the tests are running

Reply:
Just found the button! :confetti_ball: